docs/introduction/configuration/configuration.md

```diff
@@ -157,14 +157,15 @@ options:
 
 ## TypeSpec Configuration Options
 
-| Config          | Cli                       | Description                    |
-| --------------- | ------------------------- | ------------------------------ |
-| `output-dir`    | `--output-dir`            | Default output directory       |
-| `trace`         | `--trace`                 | Specify tracing area to enable |
-| `warn-as-error` | `--warn-as-error`         | Treat warning as error         |
-| `imports`       | `--import`                | Additional imports to include  |
-| `emit`          | `--emit`                  | Emitter configuration          |
-| `options`       | `--option` or `--options` | Emitter configuration          |
+| Config          | Cli                       | Description                      |
+| --------------- | ------------------------- | -------------------------------- |
+| `output-dir`    | `--output-dir`            | Default output directory         |
+| `tspconfig-dir` | `--tspconfig-dir`         | Folder containing tspconfig.yaml |
```

Comment:
I think this should be `tsp-config-dir`

Reply:
`--config-dir`?  I wouldn't expect someone to specify the filename. It looks either for `tspconfig.yaml` or `cadl-project.yaml`. Otherwise we are saying we'll take an arbitrarily named file as the config, which is totally fine... _if_ that's what we want.

Reply:
I think it's fine to allow another file name. Maybe you have tspconfig.special.yaml in same directory.

I would model it after tsc --project:

```
 --project, -p  Compile the project given the path to its configuration file, or to a folder with a 'tsconfig.json'.
```

If you pass a dir, it looks for tsconfig.json in that dir. If you pass a file it uses it, whatever it's called. But I wouldn't call it "project". So `--config <path to file or to directory that contains tspconfig.yaml>` is my vote.

Reply:
Sounds reasonable to me! @markcowl assuming you concur?

Reply:
Changed to `--config`, which can take a file path or folder.